packages/app/src/systems/Tweet/components/TwitterDialog.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from "react";
+
+import { useTwitterDialog } from "../hooks/useTwitterDialog";
+
+import { Button, Dialog, Link, useDialog } from "~/systems/UI";
+
+const tweetText = `I'm using #SwaySwap, a blazingly fast DEX built on Fuel @fuellabs_\nhttps://fuellabs.github.io/swayswap`;
```

Comment:
Can this be replaced with some placeholder random text for now? You can just comment this line so it'll be east to change later.

Reply:
Remove the entire text?

Reply:
```suggestion
// const tweetText = `I'm using #SwaySwap, a blazingly fast DEX built on Fuel @fuellabs_\nhttps://fuellabs.github.io/swayswap`;
const tweetText = `foo`;
```